docs/source/nitpick-exceptions

```diff
@@ -32,26 +32,36 @@ py:class aiida.engine.runners.ResultAndNode
 py:class aiida.engine.runners.ResultAndPk
 py:class aiida.engine.processes.workchains.workchain.WorkChainSpec
 py:class aiida.manage.manager.Manager
+py:class aiida.orm.nodes.node.WarnWhenNotEntered
 py:class aiida.orm.utils.links.LinkQuadruple
 py:class aiida.tools.importexport.dbexport.ExportReport
 py:class aiida.tools.importexport.dbexport.ArchiveData
 py:class aiida.tools.groups.paths.WalkNodeResult
 
-py:class Node
```

Comment:
This is actually something I've raised before: https://github.com/sphinx-doc/sphinx/issues/8498 (well similar). It was closed because they thought it was not an issue and I didn't get round to replying, but yeh from this I'm pretty sure it is still an issue

Reply:
so if you have the time, maybe add a comment there and I will re-open it